PhysicsTools/NanoAOD/python/nano_cff.py

```diff
@@ -151,32 +159,12 @@ def nanoAOD_addDeepBTagFor80X(process):
     process.bJetVars.src="selectedUpdatedPatJets"
     process.slimmedJetsWithUserData.src="selectedUpdatedPatJets"
     process.qgtagger80x.srcJets="selectedUpdatedPatJets"
+    if addDeepFlavour:
+        process.pfDeepFlavourJetTags.graph_path = 'RecoBTag/Combined/data/DeepFlavourV03_10X_training/constant_graph.pb'
+        process.pfDeepFlavourJetTags.lp_names = ["cpf_input_batchnorm/keras_learning_phase"]
```

Comment:
@mverzett can you please confirm that this is the right configuration for all samples (including 2016 and 2017)?